CHANGELOG.md

```diff
@@ -5,13 +5,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Added
```

Comment:
I have not checked this yet myself but wanted to ask: Does this PR require minor or major changes in any of the user guides resp. have you checked that? Imo, we should try to keep them up-to-date, even if we do not want to spend a lot of time into them currently.
Would however be fine for me to have such changes in an upcoming mini-PR.

Reply:
since this will be followed up by at least 1 more PR Iw ill shift that scan to then, apart from that a quick glance reveals that acqfs do not play a major role in the user guide yet, hence I dont expect much. I will likely add a section in the recommenders guide or make a completely new one when everything is done